src/app/modules/auth/auth-routing.module.ts

```diff
@@ -1,16 +1,22 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
-import { LoginComponent } from './components/login/login.component';
 import { ComposedLayoutComponent } from './components/composed-layout/composed-layout.component';
+import { afterLoginGuard } from 'src/app/core/guards/afterLogin/after-login.guard';
 
 const routes: Routes = [
   {
     path: '',
     component: ComposedLayoutComponent,
```

Comment:
If the component is not used (as you state down below) then you don't need to put it right here. I also think that you could clean up some unused code instead of comment it